docs/en/table_design/hybrid_table.md

```diff
@@ -29,7 +29,7 @@ As an OLAP database, StarRocks originally stores data in the columnar storage, w
 
 - The table must be a Primary Key table.
 - The length of the `__row` column cannot exceed 1 MB.
-- Columns cannot be of data types like ARRAY, MAP, and STRUCT.
+- Since v3.2.4, the column types newly support to be BITMAP, HLL, JSON, ARRAY, MAP, and STRUCT.
```

Comment:
Since v3.2.4, the supported column types are BITMAP, HLL, JSON, ARRAY, MAP, and STRUCT.
or 
v3.2.4 extends column type support to BITMAP, HLL, JSON, ARRAY, MAP, and STRUCT.